src/utils/FindUserById.tsx

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
```

Comment:
[CODE STYLE] According to the checklist, you should prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. You are importing data from the server in a non-root component. You should move this logic to the root component.